frontend/public/components/edit-yaml.jsx

```diff
@@ -18,6 +22,8 @@ import { coFetchJSON } from '../co-fetch';
 import { ResourceSidebar } from './sidebars/resource-sidebar';
 import { yamlTemplates } from '../models/yaml-templates';
 
+import withDragDropContext from './utils/drag-drop-context';
```

Comment:
last nit: lets move this import one line up to the rest of out component imports